js/src/views/Account/Header/header.js

```diff
@@ -67,6 +71,10 @@ export default class Header extends Component {
             <Balance
               account={ account }
               balance={ balance } />
+            <Certifications
+              certifications={ certifications }
+              dappsUrl={ api.dappsUrl }
```

Comment:
What I meant is that you can connect your dumb `Certifications` component to the Redux store, so the Connector won't be dumb, but the presentation Component will be. It's just not to have to pass props to it all the way from `Account` which doesn't really care about the `certifications` per se